sound/soc/intel/boards/sof_rt5682.c

```diff
@@ -1007,6 +1007,17 @@ static const struct platform_device_id board_ids[] = {
 					SOF_RT5682_SSP_AMP(2) |
 					SOF_RT5682_NUM_HDMIDEV(4)),
 	},
+	{
+		.name = "adl_mx98360a_rt5682",
+		.driver_data = (kernel_ulong_t)(SOF_RT5682_MCLK_EN |
+					SOF_RT5682_SSP_CODEC(0) |
+					SOF_SPEAKER_AMP_PRESENT |
+					SOF_MAX98360A_SPEAKER_AMP_PRESENT |
+					SOF_RT5682_SSP_AMP(1) |
+					SOF_RT5682_NUM_HDMIDEV(4) |
+					SOF_BT_OFFLOAD_SSP(2) |
```

Comment:
The commit message seems wrong

"
Follow Intel BT offload design to connect
ALC5682 to SSP0, max98360a speaker amp to SSP1, and DMIC to SSP2.
"

should it be BT_OFFLOAD to SSP2?

Reply:
DMIC is connected to SSP2 pinout while BT and SOF is using VGPIO inside PCH. Anyway, the commit message is confusing. I will revise it. Thanks.

Reply:
updated